checks/evaluation/security_policy.go

```diff
@@ -43,7 +95,17 @@ func SecurityPolicy(name string, dl checker.DetailLogger, r *checker.SecurityPol
 		} else {
 			msg.Text = "security policy detected in current repo"
 		}
+
+		var contentLen, linkedContentLen, urls, emails, discvuls int
+		fmt.Sscanf(f.Snippet, "%d,%d,%d,%d,%d", &contentLen, &linkedContentLen, &urls, &emails, &discvuls)
```

Comment:
It's fine to use multiple Info/Warn, I think.

Reply:
On this one--I am not sure what you are hinting at. I think I need more clarification on this comment/suggestion please.

Reply:
sorry. I mean that when reporting what we find, we can have `Info("found an email: %s")`, another one `Info("we found a URL: %s)`, etc. We don't need to add everything to a single snippet.

Reply:
So I did not try to log.Info() that actual content found, just the check that was satisfied (scored 10):

```
    "checks": [
        {
            "details": [
                "Info: Found linked content in security policy: SECURITY.md",
                "Info: Found text in security policy: SECURITY.md",
                "Info: Found disclosure, vulnerability, and/or timelines in security policy: SECURITY.md",
                "Info: security policy detected in current repo: SECURITY.md:1"
            ],
```

and here is an example where a site scored 0 (zero)

```
    "checks": [
        {
            "details": [
                "Warn: no email or URL found in security policy: SECURITY.md",
                "Warn: No text (beyond any linked content) found in security policy: SECURITY.md",
                "Warn: One or no descriptive hints of disclosure, vulnerability, and/or timelines in security policy: SECURITY.md",
                "Info: security policy detected in current repo: SECURITY.md:1"
            ],

```